modules/manage/partials/iceberg/use-iceberg-catalogs.adoc

```diff
@@ -39,9 +38,114 @@ For REST catalogs that use self-signed certificates, also configure these proper
 
 See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
 
+ifdef::env-cloud[]
+=== Store a secret for REST catalog authentication
+
+To store a secret that you can reference in your catalog authentication cluster properties, you must create the secret using `rpk` or the Cloud API. Secrets are stored in the secret management solution of your cloud provider. Redpanda retrieves the secrets at runtime. 
+
+To learn more about `rpk` and the Cloud API, see xref:manage:rpk/intro-to-rpk.adoc[] and xref:manage:api/cloud-api-overview.adoc[].
+
+Store secrets for the following properties:
```

Comment:
btw. not all of these are needed for all catalogs so we should make sure to word it in such way that if you need to use any of these properties, they have to be done via the secret references.

Reply:
Rephrased